pkg/tls/tls.go

```diff
@@ -309,6 +324,54 @@ func getClusterIP(services corev1controllers.ServiceController) (string, error)
 	return service.Spec.ClusterIP, nil
 }
 
+func getHostIPs(deployments appscontrollers.DeploymentController, nodes corev1controllers.NodeController) ([]string, error) {
+	deployment, err := deployments.Get(namespace.System, "rancher", metav1.GetOptions{})
```

Comment:
Any way we can consolidate the use of `rancher` here into a constant? We're also using it above on line 314.

Reply:
Done.